l10n_br_ie_search/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 KMEE
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Brazilian Localization IE Search",
+    "summary": """
+        Integração com a API SintegraWS e SEFAZ""",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "KMEE,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/l10n-brazil",
+    "depends": ["l10n_br_cnpj_search", "l10n_br_nfe"],
+    "data": ["views/res_config_settings_view.xml"],
+    "demo": [],
```

Comment:
Se não for adicionar dado de demo, pode remover essa chave. Eventualmente podemos adicionar ela caso seja adicionado.